utility/importer/brave_safari_importer.mm

```diff
@@ -0,0 +1,71 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <Cocoa/Cocoa.h>
+
+#include "brave/utility/importer/brave_safari_importer.h"
+
+#include <string>
+#include <vector>
+
+#include "base/files/file_path.h"
+#include "base/files/file_util.h"
+#include "base/strings/utf_string_conversions.h"
+#include "base/time/time.h"
+#include "chrome/common/importer/importer_bridge.h"
+#include "chrome/common/importer/importer_url_row.h"
+#include "sql/statement.h"
+#include "url/gurl.h"
+
+BraveSafariImporter::~BraveSafariImporter() = default;
+
+void BraveSafariImporter::ImportHistory() {
```

Comment:
I'm confused, why are we doing this if chrome already imports the history?

Reply:
Chrome's history importing from Safari is broken. filed issue to https://bugs.chromium.org/p/chromium/issues/detail?id=1078626
The reason is Chrome still check `History.plist` but that file is not existed anymore and `History.db` is introduced.

Reply:
Nice! This is one we can upstream maybe?

Reply:
I'll try :)

Reply:
@bridiver Do you have more concerns for this PR?